src/Casbin.AspNetCore.Core/Transformers/CustomRequestTransformer.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Casbin.AspNetCore.Authorization.Transformers
+{
+    public class CustomRequestTransformer : IRequestTransformer
```

Comment:
The build-in transformers will use the format of "{ModelName}RequestTransformer", like "RbacRequestTransformer" etc.
The model list is https://github.com/casbin/Casbin.NET#examples.
So, this class will rename to "KerMatchRequestTransformer".